integration.md

```diff
@@ -81,30 +81,32 @@ We've previously stated that the forms presented by Klarna are eventually posted
 ```ruby
 # Handle POST requests to '/purchase'
 post '/purchase' do
+  @user = User.create(session, params[:userToken], 'user@email.com') unless @user
   purchase = Klarna::Purchase.new(
-    user_token:    params[:userToken],
-    reference:     "IA-#{params[:article_id]}",
-    name:          'Interesting Article',
+    user_token:    @user.token,
+    reference:     'subscription',
```

Comment:
Yes, like Yuval said. This is meant more as a general purpose sort of thing in the spirit of the example. Will someone reading the docs know what `@user` is? No, they will not, and there's no point of confusing them with it.
Also, this is a section about one time payments. What's the point of changing this to talk about subscriptions?